store/searchlayer/channel_layer.go

```diff
@@ -151,18 +151,18 @@ func (c *SearchChannelStore) AutocompleteInTeam(teamId string, term string, incl
 		mlog.Debug("Using database search because no other search engine is available")
 		channelList, nErr = c.ChannelStore.AutocompleteInTeam(teamId, term, includeDeleted)
 		if nErr != nil {
-			return nil, model.NewAppError("AutocompleteInTeam", "app.channel.search.app_error", nil, nErr.Error(), http.StatusInternalServerError)
+			return nil, errors.Wrapf(err, utils.T("app.channel.search.app_error"))
```

Comment:
The translations are only done at the top layer when the error message is returned to the client. We don't need to do it for internal errors like these. I'd recommend: `errors.Wrapf(err, "Failed to autocomplete channels in team")`

Reply:
good to know. thanks